tools/libs/light/libxl_pci.c

```diff
@@ -81,16 +81,19 @@ static int pci_handle_response(libxl__gc *gc,
     LOG(DEBUG, "command: %s", command_name);
 
     if (strcmp(command_name, PCID_CMD_LIST_ASSIGNABLE) == 0)
-       ret = process_list_assignable(gc, response, result);
+        ret = process_list_assignable(gc, response, result);
```

Comment:
Looks like spurious change.

Reply:
I can say - strange change. I can't see this in my code